api/app/signals/settings/development.py

```diff
@@ -9,7 +9,7 @@
 
 SIGNALS_AUTHZ = {
     'JWKS_URL': "http://dex:5556/keys",
-    'ALWAYS_OK': False,
+    'ALWAYS_OK': True,
```

Comment:
Hi @boris-arkenaar ,

We want to keep this set to `False` in the development settings. We want to be able to run the complete setup including the DEX identity provider. The tester at Amsterdam side is also going to use this setup for testing.

You can always declare a `local.py` file to overwrite any settings that you need for development on your local machine. This file is ignored by git.